client/src/styles/index.css

```diff
@@ -1,7 +1,21 @@
 @import "tailwindcss/base";
 @import "tailwindcss/components";
-@import "./Button";
 @import "tailwindcss/utilities";
 
-@import "./typography.css";
+@import "./Button";
 @import "./Layout";
+
+@layer base {
+  h1 {
```

Comment:
I would maybe use something like `.title-1` etc.. so that we need to be explicit when we want to use these styles